openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/PopoverContent.tsx

```diff
@@ -195,9 +194,9 @@ const PopoverContent: FC<Props> = ({
 
       {editCheck && (
         <Icon
-          component={EditIcon}
+          component={IconEdit}
           data-testid="edit-message"
-          style={{ fontSize: '18px' }}
+          style={{ fontSize: '22px' }}
```

Comment:
Why we are using different different fontSize?

Reply:
due to the difference in size displayed on the UI, although the icons are the same in size.